CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Removed
 
 - Removed `phylum group transfer` subcommand
+- Removed owner email from `phylum group list` results
```

Comment:
```suggestion
- Owner email from `phylum group list` results
```

Forgot to mention that for the previous changelog entry, but generally we don't add `Removed` in front of every removed entry, since it's redundant.

Feel free to also change the other entry.